app/src/ui/app.tsx

```diff
@@ -2469,6 +2520,14 @@ export class App extends React.Component<IAppProps, IAppState> {
       kind: HistoryTabMode.History,
     })
   }
+
+  private inNoRepositoriesBlankSlateState: () => boolean = () => {
+    return this.state.repositories.length === 0 || this.isTutorialPaused()
+  }
+
+  private isTutorialPaused: () => boolean = () => {
```

Comment:
The same suggestion as above here that I don't see this method ever getting passed around so I don't think it needs to be bound
```suggestion
  private isTutorialPaused() {
```